ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -429,6 +429,12 @@ echo "$(date) - waiting for clusteroperators to finish progressing..."
 oc wait clusteroperators --all --for=condition=Progressing=false --timeout=10m
 echo "$(date) - all clusteroperators are done progressing."
 
+# reportedly even the above is not enough for etcd which can still require time to stabilize and rotate certs:
+echo "$(date) - waiting for oc adm wait-for-stable-cluster..."
+oc adm wait-for-stable-cluster --minimum-stable-period 2m
```

Comment:
How long has wait-for-stable-cluster been around? This step gets used across all versions of openshift

Reply:
Looks pretty new https://github.com/openshift/oc/pull/1444